src/test/java/roomescape/ReservationsTest.java

```diff
@@ -0,0 +1,44 @@
+package roomescape;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class ReservationsTest {
+    @Test
+    void constructor() {
+        // when
+        Reservations reservations = new Reservations();
+
+        // then
+        assertAll(
+                () -> Assertions.assertThat(reservations.getReservations().size()).isEqualTo(0),
```

Comment:
```suggestion
                () -> Assertions.assertThat(reservations.getReservations()).isEmpty(),
```
AssertJ는 다양한 메서드를 제공하고 있어요~

Reply:
까먹고 있었네요! 감사합니다 꼼꼼한 피드백 감사합니다 :)